### DISCUSSION TITLE
How do I remove PWA?

### DISCUSSION THREAD

**Discussion by Owslebury** [cotes2020/jekyll-theme-chirpy#1481] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I'd like my website to not have an option that lets you install an app.
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/121823795/6cca8a42-7c82-4b9d-9ce5-8e549c4d081a)

### Operations you have already tried

I tried changing the PWA option in the config file to false and then served the site both online and local and it's still there. Any help appreciated.

**Comment by cotes2020** [comment, marked as answer]
~~Removing `assets/js/pwa/unregister.js`~~

Will be fixed in #1501
